packages/create-app/src/utils/template/npm.ts

```diff
@@ -21,30 +21,11 @@ export async function updateCLIDependencies({
   packageJSON.dependencies['@shopify/app'] = cliKitVersion
 
   if (local) {
-    // CLI path
-    const cliAbsolutePath = (await path.findUp('packages/cli-main', {
-      type: 'directory',
-      cwd: moduleDirectory,
-    })) as string
-    const cliPath = `file:${cliAbsolutePath}`
-
-    // App path
-    const appAbsolutePath = (await path.findUp('packages/app', {type: 'directory', cwd: moduleDirectory})) as string
-    const appPath = `file:${appAbsolutePath}`
-
-    // CLI Kit path
-    const cliKitAbsolutePath = (await path.findUp('packages/cli-kit', {
-      type: 'directory',
-      cwd: moduleDirectory,
-    })) as string
-    const cliKitPath = `file:${cliKitAbsolutePath}`
-
-    // UI Extensions CLI path
-    const extensionsCliAbsolutePath = (await path.findUp('packages/ui-extensions-cli', {
-      type: 'directory',
-      cwd: moduleDirectory,
-    })) as string
-    const extensionsCliPath = `file:${extensionsCliAbsolutePath}`
+    const cliPath = await packagePath('cli-main')
+    const appPath = await packagePath('app')
+    const cliKitPath = await packagePath('cli-kit')
+    const uiExtensionsCliPath = await packagePath('ui-extensions-cli')
+    const pluginNgrokPath = await packagePath('plugin-ngrok')
```

Comment:
Nice catch! If we continue adding packages to the monorepo we might want to evolve this implementation to get the list of packages using some globbing.